internal/util/indexparamcheck/conf_adapter_mgr.go

```diff
@@ -46,6 +46,12 @@ func (mgr *ConfAdapterMgrImpl) GetAdapter(indexType string) (ConfAdapter, error)
 }
 
 func (mgr *ConfAdapterMgrImpl) registerConfAdapter() {
+	mgr.adapters[IndexFaissGPUIDMap] = newBaseConfAdapter()
+	mgr.adapters[IndexFaissGPUIvfFlat] = newIVFConfAdapter()
+	mgr.adapters[IndexFaissGPUIvfPQ] = newIVFPQConfAdapter()
+	mgr.adapters[IndexFaissGPUIvfSQ8] = newIVFSQConfAdapter()
+	mgr.adapters[IndexRaftIvfFlat] = newIVFConfAdapter()
+	mgr.adapters[IndexRaftIvfPQ] = newIVFPQConfAdapter()
```

Comment:
Need double check with @Presburger to see if configs for RAFT/FAISS are the same

Reply:
Also why we don't have RAFT IVF SQ/BF?

Reply:
RAFT only support "RAFT_FLAT" and "RAFT_PQ" now